### PR TITLE
Change SelectMultiple tests to improve run time

### PR DESCRIPTION
**Pull request by jcfilben**
#### What does this PR do?
Improve run time of SelectMultiple tests to avoid timeout issues

#### Do Jest tests follow these best practices?

- [ ] `screen` is used for querying.
- [ ] The correct query is used. (Refer to [this list of queries](https://testing-library.com/docs/queries/about/#priority))
- [ ] `asFragment()` is used for snapshot testing.

#### Do the grommet docs need to be updated?
no
#### Should this PR be mentioned in the release notes?
no
#### Is this change backwards compatible or is it a breaking change?
backwards compatible

### CODE REVIEW

File: src/js/components/SelectMultiple/__tests__/SelectMultiple-test.tsx [hunk 1]
```diff
@@ -23,9 +23,67 @@ const TestOnSearch = () => {
...
+        value={value}
+        onChange={({ selected }) => setValue(selected)}
+      />
+    </Grommet>
+  );
+};
+
 describe('SelectMultiple', () => {
-  window.scrollTo = jest.fn();
-  beforeEach(createPortal);
```
Comment (taysea) [inline src/js/components/SelectMultiple/__tests__/SelectMultiple-test.tsx:28]: Is this no longer needed?

File: src/js/components/SelectMultiple/__tests__/SelectMultiple-test.tsx [hunk 2]
```diff
@@ -555,3 +469,83 @@ describe('SelectMultiple', () => {
     expect(screen.queryByRole('option', { name: /c selected/ })).not.toBeNull();
   });
 });
+
+describe('SelectMultiple with portal', () => {
+  window.scrollTo = jest.fn();
+  beforeEach(createPortal);
```
Comment (taysea) [inline src/js/components/SelectMultiple/__tests__/SelectMultiple-test.tsx:475]: Can you remind me what this is doing?

File: src/js/components/SelectMultiple/__tests__/SelectMultiple-test.tsx (hunk 1 above)
Comment (taysea) [reply src/js/components/SelectMultiple/__tests__/SelectMultiple-test.tsx:28]: Ah I see this moved to the next block below.

File: src/js/components/SelectMultiple/__tests__/SelectMultiple-test.tsx [hunk 3]
```diff
@@ -379,49 +348,17 @@ describe('SelectMultiple', () => {
...
-            onChange={({ value }) => {
-              setValueMultiple(value);
-            }}
-          />
-        </Grommet>
-      );
-    };
-    render(<Test />);
+  test('search with select and clear', () => {
+    render(<TestManyOptions />);
```
Comment (taysea) [inline src/js/components/SelectMultiple/__tests__/SelectMultiple-test.tsx:352]: Could you elaborate on how pulling this out helps with timing? I've seen the pattern before and in support, but didn't realize it affect timing.

**Review by taysea — APPROVED** [review]
Just some small questions for my own knowledge, but this looks good.

**Comment by taysea** [comment]
> Just some small questions for my own knowledge, but this looks good.

Actually one comment, I'm not following the chromatic change -- can't see the difference

File: src/js/components/SelectMultiple/__tests__/SelectMultiple-test.tsx (hunk 2 above)
Comment (jcfilben) [reply src/js/components/SelectMultiple/__tests__/SelectMultiple-test.tsx:475]: This allows us to take snapshots of the drop

File: src/js/components/SelectMultiple/__tests__/SelectMultiple-test.tsx (hunk 3 above)
Comment (jcfilben) [reply src/js/components/SelectMultiple/__tests__/SelectMultiple-test.tsx:352]: Yeah this was just to make the readability of the tests a little cleaner, it doesn't affect timing

**Review by britt6612 — APPROVED** [review]

**Pull request merged**